dk-installer.py

```diff
@@ -1653,23 +1675,29 @@ def _delete_project(self, args):
             raise AbortAction
         else:
             if not args.keep_config:
-                delete_file(DOCKER_COMPOSE_FILE)
+                delete_file(get_docker_compose_path())
             delete_file(CREDENTIALS_FILE.format(args.prod))
             CONSOLE.msg("Docker containers and volumes deleted")
 
     def _delete_network(self):
-            try:
-                self.run_cmd(
-                    "docker",
-                    "network",
-                    "rm",
-                    DOCKER_NETWORK,
-                    raise_on_non_zero=True,
-                )
-            except CommandFailed:
-                LOG.info(f"Could not delete Docker network '{DOCKER_NETWORK}'")
-            else:
-                CONSOLE.msg("Docker network deleted")
+        start_time = time.time()
+        try:
+            self.run_cmd(
+                "docker",
+                get_docker_compose_path(),
+                "network",
+                "rm",
+                DOCKER_NETWORK,
+                timeout=60,
```

Comment:
When a timeout happens, the `subprocess.TimeoutExpired` exception is raised, and it shouldn't be needed to measure the elapsed time. Also, it's probably not working as you expect exactly because this exception will prevent  code bellow the `run_cmd()` to execute. You will probably need to use the `start_cmd()` instead. There's an example of this approach for pulling the Testgen images.

Also, could you add a constant for the timeout period?